subdomain: properly check that a DNS zone has no more records inside

zaibon opened the pull request:
fixes #28

zaibon committed:
subdomain: properly check that a DNS zone has no more records inside

fixes #28

LeeSmet commented on dns/dns_zone.go:
```diff
@@ -86,6 +86,15 @@ func (z *Zone) Remove(r Record) {
+func (rs records) IsEmpty() bool {
+	for _, records := range rs {
+		if len(records) > 0 {
+			return true
```
s/true/false ?

zaibon replied:
woopsy

LeeSmet reviewed (changes requested):
code looks good but tests are failing for some reason

zaibon committed:
update dependencies

github.com/threefoldtech/tfexplorer => v0.4.0
github.com/threefoldtech/zos => v0.4.2
